hazelcast-client/src/main/java/com/hazelcast/client/impl/ClientDynamicClusterConfig.java

```diff
@@ -335,6 +338,20 @@ public Config addListenerConfig(ListenerConfig listenerConfig) {
     }
 
     @Override
+    public Config addEventJournalConfig(EventJournalConfig eventJournalConfig) {
+        String mapName = eventJournalConfig.getMapName();
+        String cacheName = eventJournalConfig.getCacheName();
+        if (StringUtil.isNullOrEmpty(mapName) && StringUtil.isNullOrEmpty(cacheName)) {
```

Comment:
Isn't one of them being not null was enough ? This look like it should have been ||, right?
And the message is confusing in exception. 
`Event journal config should have non-empty map name and/or cache name`
Is it `and` or `or` ? According to code it should be just `and`